### ISSUE TITLE
Configuration window overly wide

### ISSUE THREAD

**Issue by edakiri** [mumble-voip/mumble#2120]
The configuration window is overly and unnecessarily wide.
This causes it to not fit on my screen when it is showing the German language.
Looking at each pane in that window, I do not see anything on any pane which should prevent it from being displayed more narrowly. Some panes have much extra space.
To start in German from BASh,
`LANG=de_DE.UTF-8 mumble`

**Comment by Tarun80** [comment]
I disagree. Perhaps you don't have Advanced checked? Some things are actually not wide enough to see everything like in Messages. Having it as is allows easier, better fine tuning for settings with slider bars.

Audio Input:
![Audio Input - Advanced](https://i.imgur.com/uYwCIHw.png)

Messages:
![Messages - Advanced](https://i.imgur.com/6OVZWMG.png)

**Comment by edakiri** [comment]
You did not run it in German.
There is a scalability problem in what you showed, perhaps it is related.
Here is the Configuration->Plugins window at minimum size.
Observe that the right hand side is clipped off because it is beyond the edge of the screen.

**Comment by Tarun80** [comment]
That happens with mine as well, though I have more plugins listed. Perhaps the checklist boxes need to be more fluid in having their columns resized.

![Plugins](https://i.imgur.com/qrP3D60.png)
